src/app/ui/members-quest-approval/members-quest-approval.component.ts

```diff
@@ -35,13 +38,18 @@ export class MembersQuestApprovalComponent implements OnInit {
     private dialog: MatDialog,
     private notify: NotifyService,
     private playerQuest: PlayerQuestService,
-    private season: SeasonService) {}
+    private season: SeasonService,
+    private teamsService: TeamsService) {}
 
   async ngOnInit() {
     if (this.user && this.user.membership)  {
-      const teamId = this.user.membership.teamId;
-      const seasonId = await this.season.getEnabledSeasonId().toPromise();
-      this.questsForApproval$ = this.playerQuest.getAllMemberSubmittedQuests(seasonId, teamId);
+      this.seasonId = await this.season.getEnabledSeasonId().toPromise();
+      this.leadMemberships$ = this.teamsService.getAllLeadMemberships(this.user.uid)
```

Comment:
You can also default membership from user object and seasonId from auth, so this can be like: 
```suggestion
      this.selectedLeadMembership = this.user.membership.teamId;
      this.leadMemberships$ = this.teamsService.getAllLeadMemberships(this.user.uid);
      this.questsForApproval$ = formControlName.valueChanges.pipe(
        flatMap((teamId: string) => this.getAllMemberSubmittedQuests(this.auth.seasonId, teamId))
      );
```
you can remove `changeSelectedMembership` with this

Reply:
the selectedLeadMembership was removed in place of formControl